ibridgesgui/config.py

```diff
@@ -131,6 +131,21 @@ def _get_config() -> Union[None, dict]:
         print(f"CANNOT START APP: {CONFIG_FILE} incorrectly formatted.")
         sys.exit(1)
 
+def save_current_settings(env_path_name: Path):
+    """Store the environment with the currently scrambled password in irodsA."""
+    with open(IRODSA, "r", encoding="utf-8") as f:
+        pw = f.read()
+    config = _get_config()
+    if config is not None:
+        if "settings" not in config:
+            config["settings"] = {}
+        config["settings"][str(env_path_name)] = pw
+        _save_config(config)
+
+def get_prev_settings():
+    """Extract the settings from the configuration."""
+    config = _get_config()
```

Comment:
You will get an error if `_get_config` returns None, right? Might want to handle that case.

Reply:
Done!